hello-world/src/worker/index.ts

```diff
@@ -1,7 +1,5 @@
-'use strict';
-
-// @@@SNIPSTART nodejs-js-hello-worker
-const { Worker } = require('@temporalio/worker');
+// @@@SNIPSTART nodejs-hello-worker
```

Comment:
Can you please copy all of the comments too? They're important for first time users and the snippets.
This goes for all of the files.